analysis_file/lib/fold_data.py

```diff
@@ -0,0 +1,99 @@
+import time
+
+from core_data_modules.traced_data import Metadata
+
+
+class FoldData(object):
```

Comment:
I'm going to want to re-review this code after it has tests and has been moved into core

Reply:
I know :)